pkgs/development/python-modules/python-arango/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, pyjwt
+, requests
+, requests-toolbelt
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "python-arango";
+  version = "7.5.0";
+  disabled = pythonOlder "3.7";
```

Comment:
```suggestion
  disabled = pythonOlder "3.7";
  format = "setuptools";
```

Reply:
Done! Thank you! Again, didn't realize this existed either.